x/gamm/simulation/sim_msgs.go

```diff
@@ -214,19 +214,26 @@ func RandomJoinSwapExternAmountIn(k keeper.Keeper, sim *simtypes.SimCtx, ctx sdk
 		return nil, fmt.Errorf("no sender with denoms %s exists", coinIn.Denom)
 	}
 
-	// cap joining pool to the pool liquidity
-	newTokenIn := osmoutils.MinCoins(sdk.NewCoins(coinIn), sdk.NewCoins(tokenIn))
+	r := sim.GetSeededRand("select random seed")
```

Comment:
Let's define the seed as a constant at the top of the file for visibility please

Reply:
this is just a variable declaration right? I think it's fine to leave it here, which is right before the actual usage of the varaible